### DISCUSSION TITLE
Please how do I go about my GitHub student development pack

### DISCUSSION THREAD

**Discussion by davidakaluzia** [community/community#145001] [GitHub Education]
### Hi everyone,

Hello am new to the community, please how do I go about my student verification.

**Comment by Somkr999** [comment]
You just need an e-mail issued by your college. If not then some documents preferably id card it will then pin your location to see if you are in the same city as your college and done. You still need to wait 1-2 days before the confirmation, if there are any problems with your application you will be notified immediately you can go ahead correct them and reapply.

**Reply by davidakaluzia** [reply]
Okay, my student ID card
